app/Http/Controllers/EventController.php

```diff
@@ -90,7 +91,37 @@ public function show($id, Request $request)
      */
     public function update(Request $request, $id)
     {
-        return  response()->json(['status' => 'error', 'message' => 'method_not_implemented'], 501);
+        $event = Event::find($id);
+
+        if (isset($request->organizer)) {
+            $organizer = UserController::getUserByIdentifier($request->organizer);
+            $request['organizer'] = $organizer->id;
+        }
+
+        
+        $this->validate($request, [
+            'name' => 'required|max:255',
+            'price' => 'numeric',
```

Comment:
Will numeric allow for decimals? I don't know why it wouldn't, but just checking in case prices are not round dollar amounts.

Reply:
Yes, this allows decimals.

The `Integer` validator rule is used otherwise.